src/main/groovy/de/felixschulze/gradle/HockeyAppUploadTask.groovy

```diff
@@ -147,11 +153,13 @@ class HockeyAppUploadTask extends DefaultTask {
             }
         }
 
-        uploadAppplicationFileToHockeyApp(applicationFile, mappingFile, appId)
+        applicationFiles.each {
+            uploadApplicationFileToHockeyApp(it, mappingFile, appId)
```

Comment:
I opened a issue on HockeyApp about this: http://support.hockeyapp.net/discussions/problems/49508-api-upload-apk-split-version